components/webui/server/src/fastify-v2/plugins/app/socket/MongoSocketIoServer/index.ts

```
@@ -301,26 +311,26 @@ class MongoSocketIoServer {
+        this.#logger.info(`Socket:${socket.id} unsubscribed from query:${queryId}`);
 
         if (isLastSubscriber) {
-            this.#fastify.log.debug(`Query:${queryId} deleted from query map.`);
+            this.#logger.debug(`Query:${queryId} deleted from query map.`);
             this.#queryIdToQueryHashMap.delete(queryId);
         }
 
-        this.#fastify.log.debug(
+        this.#logger.debug(
             "Query ID to query hash map:" +
             ` ${JSON.stringify(Array.from(this.#queryIdToQueryHashMap.entries()))}`
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)